src/Compilers/CSharp/Portable/Binder/Binder_Crefs.cs

```diff
@@ -358,6 +359,16 @@ private ImmutableArray<Symbol> ComputeSortedCrefMembers(NamespaceOrTypeSymbol? c
                     builder.AddRange(result.Symbols);
                     result.Free();
                 }
+                else if (memberName is "nint" or "nuint"
```

Comment:
I feel like `@nint` should still never bind to native integers, to match how predefined type keywords do. That would make the language more consistent.

Reply:
I think you're right. I should have kept that part